frontend/public/style/_layout.scss

```diff
@@ -16,6 +16,7 @@ body,
 #content-scrollable {
   overflow: auto;
   height: 100%;
+  position: relative;
```

Comment:
Is this really needed?

The Edit form doesn't need this yet, so why is this now required?

/hold

Reply:
Thanks for removing this new style.

/unhold